daemon/start.go

```diff
@@ -14,6 +17,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	ContainerdImageNameLabel   = "containerd.io/image.name"
```

Comment:
Can you add a doc string for what these labels are used for?
Also are these moby labels or are they labels recognized by containerd?

Reply:
These are intended to be used by containerd.

Reply:
Since these were intended to be used as part of 38738 I'm going to remove the label addition here and add it to that PR to keep this change small with just the create opts.